_README.md

```diff
@@ -0,0 +1,145 @@
+-->
+
+Below table contains information about what parts of the specification are required by this template so it generates proper output.
+
+Property name | Reason | Fallback | Default
+---|---|---|---
+`components.schemas` | This template supports only schemas that have unique and human-readable names. Such names can also be provided if schemas are described under `components.schemas` and each schema is a separate object with its unique key. | - | -
+
+## Supported protocols
+
+<!--  
+Specify what protocols is your code generator supporting. This information should match the information provided in the template configuration under the `supportedProtocols` property. Don't put this section in your readme if your template doesn't generate code.
+-->
+
+_This template doesn't generate code and therefore is not limited to any specific protocols_
+
+## How to use the template
+
+<!--  
+Make sure it is easy to try out the template and check what it generates. Instructions for CLI and Docker should be easy to use; just copy/paste to the terminal. In other words, you should always make sure to have ready to use docker-compose set up so the user can quickly check how generated code behaves.
+-->
+
+This template must be used with the AsyncAPI Generator. All available options on how to generate output are available [here](https://github.com/asyncapi/generator/).
+
+### CLI
+
+```bash
+npm install -g @asyncapi/generator
+ag https://raw.githubusercontent.com/asyncapi/generator/v1.0.0-rc.15/test/docs/dummy.yml https://github.com/asyncapi/template-for-generator-templates -o output
+open output/index.html
+```
+
+### Docker
+
+```bash
+docker run --rm -it \
+-v ${PWD}/output:/app/output \
+asyncapi/generator -o /app/output https://raw.githubusercontent.com/asyncapi/generator/v1.0.0-rc.15/test/docs/dummy.yml @asyncapi/markdown-template --force-write
+```
+
+## Template configuration
+
+<!--  
+This information should match the information provided in the template configuration under the `parameters` property.
+-->
+
+You can configure this template using different parameters. You pass them like this in the Generator CLI: `-p PARAM1_NAME=PARAM1_VALUE -p PARAM2_NAME=PARAM2_VALUE`
+
+
+Name | Description | Required | Default | Allowed Values | Example
+---|---|---|---|---|---
+asyncapiFileDir | Custom location of the AsyncAPI file that you provided as an input in a generation. | No | The root of the output directory. | Path to the custom directory relative to the root of output directory provided as a string | `/custom/dir`
+pdf | Generate PDF from initially generated HTML file | No | `false` | `true`/`false` | `true`
```

Comment:
```suggestion
pdf | Generate PDF from initially generated HTML file. | No | `false` | `true`/`false` | `true`
```
If it's a description of a parameter, maybe it is better to use  third person singular, as in `generates` 🤔